### PR TITLE
Announces CFP and Specialist Tracks via news post.

### PR DESCRIPTION
**Pull request by devjack**

### CODE REVIEW

**Comment by devjack** [comment]
@jeamland / @excitedleigh PR for news.
@jeamland please note attribution to you.

**Review by jeamland — APPROVED** [review]

File: data/News/2020-06-10-specialist-tracks-call-for-proposals.md [hunk 1]
```diff
@@ -0,0 +1,50 @@
+---
+title: Specialist Tracks + Call for Proposals
+published: 2020-06-17
+ytid: pWyTTH44TAs
+---
+
+Starting today, PyConline AU’s 2020 call for proposals (CFP) will be open for submissions running all month until **12 July, anywhere on earth (UTC+12)**. Alongside this announcement, we’re also officially publishing our specialist tracks.
+
+## Specialist Tracks
```
Comment (daisylb) [inline data/News/2020-06-10-specialist-tracks-call-for-proposals.md]:
```suggestion
## Specialist tracks
```

File: data/News/2020-06-10-specialist-tracks-call-for-proposals.md [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+* Security and Privacy
+* Education
+* Data Science & Analytics
+* DevOops: Learning from Failure
+
+You can learn more about the specialist tracks and their topics on our updated program page at [https://2020.pycon.org.au/program](https://2020.pycon.org.au/program).
+
+## Call for proposals (CFP)
+Today we are also opening our CFP!
+Submissions to speak at PyConline AU 2020 will be accepted from now until **12 July, anywhere on earth (UTC+12)**. For those of you in Adelaide Australia that is 9:30pm the day after.
```
Comment (daisylb) [inline data/News/2020-06-10-specialist-tracks-call-for-proposals.md]:
```suggestion
Submissions to speak at PyConline AU 2020 will be accepted from now until **12 July, anywhere on earth (UTC+12)**. For those of you in Adelaide, Australia that is 9:30pm the day after.
```

File: data/News/2020-06-10-specialist-tracks-call-for-proposals.md [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+
+
+We welcome submissions from everybody, including those:
+who have never given a conference talk before,
+who are new to Python,
+who have built interesting things in Python, 
+who have helped build Python, and
+who have expertise to share with our community from fields outside of Python (and outside of technology).
```
Comment (daisylb) [inline data/News/2020-06-10-specialist-tracks-call-for-proposals.md]:
```suggestion
We welcome submissions from everybody, including those:

* who have never given a conference talk before,
* who are new to Python,
* who have built interesting things in Python, 
* who have helped build Python, and
* who have expertise to share with our community from fields outside of Python (and outside of technology).
```

**Review by daisylb — COMMENTED** [review]
Minor formatting fixes, lgtm otherwise

File: data/News/2020-06-10-specialist-tracks-call-for-proposals.md [hunk 4]
```diff
@@ -0,0 +1,51 @@
...
+
+Please also help us spread the word about our CFP by sharing a link to this page with the hashtag #PyConlineAU.
+
+To discuss this announcement and the CFP in more detail, please join us this evening at **7:30pm ACST 17 June 2020** on the PyCon AU YouTube channel: [https://youtube.com/pyconau](https://youtube.com/pyconau) (or view the embedded video above).
+
+<p>
+    Stay safe,<br>
+    <b>Benno Rice</b><br>
+    he/him<br>
+    CFP Committee Chair<br>
```
Comment (daisylb) [inline data/News/2020-06-10-specialist-tracks-call-for-proposals.md]:
```suggestion
    Program Chair<br>
```

**Review by daisylb — APPROVED** [review]

**Pull request merged**